File: Java/StartThread.java
public class StartThread extends Thread {
    public volatile boolean running = true;
    private final int id;
    private final int step;
    private final int millis;

    public StartThread(int id, int step, int millis) {
        this.id = id;
        this.step = step;
        this.millis = millis;
    }

    public int getID() {
        return id;
    }

    public int getMillis() {
        return millis;
    }

    @Override
    public void run() {
        int sum = 0;
        int count = 0;

        while (running) {
            sum += step;
            count++;
        }

        System.out.println("ID Thread " + id + ": sum = " + sum + ", count = " + count);
    }
}

File: C#/StartThread.cs
using System;

namespace LB1
{
    class StartThread
    {
        public int Id { get; }
        private int Step { get; }
        public int Millis { get;}

        public volatile bool Running = true;

        public StartThread(int id, int step, int millis)
        {
            Id = id;
            Step = step;
            Millis = millis;
        }

        public void Run()
        {
            int sum = 0;
            int count = 0;

            while (Running)
            {
                sum += Step;
                count++;
            }

            Console.WriteLine($"ID Thread {Id} sum = {sum} count = {count}");
        }
    }
}
